core-feature-pack/common/src/main/resources/content/bin/domain.ps1

```diff
@@ -44,7 +44,11 @@ foreach($p in $HOST_CONTROLLER_OPTS){
 		$JBOSS_LOG_DIR=$p.Substring('-Djboss.domain.log.dir='.Length)
 	}elseif ($arg.StartsWith('-Djboss.domain.config.dir')){
 		$JBOSS_CONFIG_DIR=$p.Substring('-Djboss.domain.config.dir='.Length)
-	}
+	}elseif ($arg.StartsWith('--host-config')){
+    	$CONFIG_FILE=$p.Substring('--host-config='.Length)
+    }elseif ($arg.StartsWith('--domain-config')){
+    	$CONFIG_FILE=$p.Substring('--domain-config='.Length)
```

Comment:
Hi @khermano Independently of the final implementation, in the server code or in the scripts, I guess this won't work as expected since you can use both options `--host-config` and `--domain-config` together to start the domain.

I've also asked to get a wider description about what is the goal of this enhancement in the Jira tickethttps://issues.redhat.com/browse/WFCORE-4867 . The intention is still a bit unclear to me. I don't see yet the utility of just printing the used config file name when you are starting the server. feel free to share any feedback about its utility in the Jira, thanks!

Reply:
Hello @yersan. Thank you for comment I will take a look at that domain situation as soon as I contact Rostislav as I claim in JIRA comment. And sorry about the late response, I somehow managed to missed it completely.